server/src/routes/index.js

```diff
@@ -0,0 +1,16 @@
+
+import restRoutes from './rest';
+
+const init = (server) => {
+  server.get('*', (req, res, next) => {
+    console.log(`Request received for ${req.originalUrl}`);
```

Comment:
@piyushagwl007 it's not a good idea to log like this when code goes for production. If you want to log for debug, try using this
https://expressjs.com/en/guide/debugging.html

Reply:
Sure will use debug console